go.mod

```
@@ -0,0 +1,19 @@
+module cataloggen
```

Since this `go.mod` file is for the entire repo, perhaps we should give it a more general name so that it won't be wrong when we add more tools here.
How about
```suggestion
module operatorindex
```